media/css/mozorg/home/home-promo.less

```diff
@@ -1066,6 +1066,55 @@ html[lang|="en"] {
 
 
 /****************************************************************************/
+// End of year fundraising
+.promo-large-landscape.fundraiser {
+    background-color: #e63d2f;
+    color: #fff;
+
+    h2 {
+        padding-top: 30px;
+        .font-size(32px);
+    }
+
+    h2:before {
+        display: block;
+        content: '';
+        .at2x('/media/img/home/voices/promos/fundraising/fox.png', 134px, 115px);
+        background-repeat: no-repeat;
+        margin: 0 auto;
+        width: 134px;
+        height: 115px;
+    }
+
+    h3 {
+        padding-top: 90px;
+    }
+
+    &.show {
+        a.panel-link {
+            border: none;
+            transition: visibility 0s ease-in-out 0s
```

Comment:
@alexgibson I am guessing this is what you are referring to? I guess I am also missing the `-webkit-` prefix version here. Also, it used to have a border transition but, as I am setting border to `none` I removed that.